source/outline.rst

```diff
@@ -1,8 +1,8 @@
-===================================================================
-Computer Science 161 -- Introduction to Programming
-===================================================================
+=====================================================================
```

Comment:
Changes in this file are unrelated to the main content of this PR. Revert (most of?) these, and open a new PR for them.

Reply:
Seems like something went wrong with a merge.

Reply:
nerp, branched off branch by accident.